BraveSharedTests/NSURLExtensionsTests.swift

```diff
@@ -6,6 +6,29 @@ import UIKit
 import XCTest
 @testable import Shared
 
+private extension String {
+  var isIPv6: Bool {
+    return self.contains(":") ?? false
```

Comment:
Isn't `https://test:8080/test` containing `:` while not being an `IPv6` at the same time?

Reply:
It's Firefox code and isn't new. But also, it's only checking if the host contains ":" and not the entire URL